### DISCUSSION TITLE
Support forward slash in fuzzy-finder etc. on Windows

### DISCUSSION THREAD

**Discussion by helmesjo** [helix-editor/helix#10202] [Ideas]
Hi! I tried to look through the issues but didn't see this one mentioned.

Basically, on Windows when using the fuzzy-finder or opening a new file etc., then one must use backslashes. It would be a great convenience if it worked with forwardslash as well (Windows supports this).

Thanks!

**Comment by helmesjo** [comment]
I might be wrong, but isn't this a trivial change given that Windows deals with forward-slash all fine? So I assume this is just some `if(windows)` somewhere in helix...
Reason for the "bump" is that I'm unfortunate enough to work on Windows, so have WSL running at all times and even this small difference in behaviour gets mentally exhausting quite quickly. 

Thanks. 🙏

**Comment by JackMordaunt** [comment]
@helmesjo checkout my PR #6564

If you are bothered by this as much as I am, build that commit and enjoy a less painful experience :)

**Reply by JackMordaunt** [reply]
For what it's worth, I no longer include slashes at all, I just fuzzy search the names within the path.
